.github/workflows/ci.yml

```diff
@@ -47,15 +47,11 @@ jobs:
   compile_and_test:
     strategy:
       matrix:
-        flink: [ 1.17.1, 1.18.0 ]
-        jdk: [ 8, 11, 17 ]
-        exclude:
-          - jdk: 17
-            flink: 1.17.1
+        flink: [ 1.18.1, 1.19.0 ]
     uses: apache/flink-connector-shared-utils/.github/workflows/ci.yml@ci_utils
     with:
       flink_version: ${{ matrix.flink }}
-      jdk_version: ${{ matrix.jdk }}
+      jdk_version: ${{ matrix.jdk || '8, 11, 17' }}
```

Comment:
Should we add `21` for 1.19?

Reply:
I have not tested support, would rather decouple this